script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt

```diff
@@ -76,7 +76,7 @@ Like nothing I've ever seen before (it's not something you'd normally see)
 An impossible sight―――
 <Page20>
                                     A melting dog.
-What is that?!
+What is that?! //Even for なんだアレは, the exclamation point seems a little strong. Thoughts?
```

Comment:
Looking back at it, I don't have a super strong preference for what punctuation to use here. Anyone else have an opinion? Might just leave as is otherwise.